test/yapay_ex_test.exs

```diff
@@ -6,9 +6,7 @@ defmodule YapayExTest do
 
   describe "create_order/1" do
     setup do
-      bypass = Bypass.open()
-      endpoint = yapay_endpoint(bypass)
-      System.put_env("YAPAY_DOMAIN", endpoint)
+      bypass = Bypass.open(port: 1337)
```

Comment:
I think you tried to specify a port here to simplify the test setup, removing the env var, am I right?

The problem with this approach is that you lost the parallelism ability of bypass. It sets a different port to different requests this way you can run your tests in parallel. If you specify a port, all stubs will run into this port and if don't mark your tests as `async: false`, you could have one stub interfering with another.

Reply:
I see! I will try to change this 🥇